.github/workflows/ci-sql-cli.yaml

```diff
@@ -26,10 +26,39 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
+# This list should only have non-sensitive env vars
+# Env vars with secrets should be in the specific jobs
+env:
+  GOOGLE_CLOUD_PROJECT: astronomer-dag-authoring
+  POSTGRES_HOST: postgres
+  POSTGRES_PORT: 5432
+  REDSHIFT_DATABASE: dev
+  REDSHIFT_HOST: utkarsh-cluster.cdru7mxqmtyx.us-east-2.redshift.amazonaws.com
+  SNOWFLAKE_SCHEMA: ASTROFLOW_CI
+  SNOWFLAKE_DATABASE: SANDBOX
+  SNOWFLAKE_WAREHOUSE: DEMO
+  SNOWFLAKE_HOST: https://gp21411.us-east-1.snowflakecomputing.com
+  SNOWFLAKE_ACCOUNT: gp21411
+  SNOWFLAKE_REGION: us-east-1
+  SNOWFLAKE_ROLE: AIRFLOW_TEST_USER
+
 jobs:
   Type-Check:
     if: github.event.action != 'labeled'
     runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: dimberman/pagila-test
```

Comment:
Just like here - I wonder if we could use the standard pagila image

Reply:
For consistency, I kept the same image.